src/binary-search-tree.js

```diff
@@ -2,6 +2,23 @@
 /* eslint-disable global-require */
 /* eslint-disable no-unused-vars */
 /* eslint-disable no-trailing-spaces */
+class Queue {
```

Comment:
This should definitely be imported instead of just pasting the code inside the file.